test/deps.test.js

```diff
@@ -82,3 +82,14 @@ test('deps - yarn with renamed dep', function (t) {
     t.fail(e.stack);
   }).then(t.end);
 });
+
+// See test/fixtures/yarn-link-deps/README.md
+test('deps - yarn with linked deps', function (t) {
+  deps('test/fixtures/yarn-link-deps').then(function (res) {
+    t.equal(res.name, 'yarn-link-deps', 'package name matches');
+    t.type(res.dependencies, 'object', 'has dependencies');
```

Comment:
I think that would be nice if you could change assertion on `has dependencies`.
Because if `res.dependencies` returns a `null or {}` it will be considered as an `object` which would result in 0 dependencies, that would make `has dependencies` results an invalid statement.

Reply:
but then next line asserts the number of dependencies.

Reply:
I understand what you mean, I just think that would be nice if we could validate the content of the dependencies e.g. are we retrieving dependencies for real?